### ISSUE TITLE
On opening, cursor points to the same file, not the last used

### ISSUE THREAD

**Issue by ruslansagitov** [scrooloose/nerdtree#1034]
#### Self-Diagnosis
- [x] I have searched the [issues](https://github.com/scrooloose/nerdtree/issues) for an answer to my question.
- [x] I have reviewed the NERDTree documentation. `:h NERDTree`
- [x] I have reviewed the [Wiki](https://github.com/scrooloose/nerdtree/wiki).
- [x] I have searched the web for an answer to my question.

#### Environment (for bug reports)
- [x] Operating System: Fedora 30
- [x] Vim/Neovim version `:echo v:version`: 8.1.1912 (801)
- [x] NERDTree version, found on 1st line in NERDTree quickhelp `?`: 5.2.9, 5.3.0
- [x] vimrc settings
    - [x] NERDTree variables
    ```vim
    ```
    - Other NERDTree-dependent Plugins
        - [x] Others (specify): none
    - [x] I've verified the issue occurs with only NERDTree installed.

#### Steps to Reproduce the Issue
1. Open NERDTree
2. Select file, open
3. Do steps 1-2 several times
4. User Ctrl+^ several times
5. Repeat steps 1-4
6. At some point of time, the cursor is always pointing to the same file, not the last opened

#### Current Result (Include screenshots where appropriate.)

On opening NERDTree, the current file cursor always moves to the same file.

#### Expected Result

On opening NERDTree, the current file cursor points to the last opened file.

#### Regression

Can be reproduced with 5.2.9 or above.

Cannot be reproduced with 5.2.8 or below.

**Comment by PhilRunninger** [comment]
@ruslansagitov I cannot reproduce this bug. Do you have any autocmds or other plugins that would interfere with this and alter the value in the alternate buffer register - `#`?

**Comment by ruslansagitov** [comment]
Sorry for delay. I was kinda busy.

So, right now, I tried again to reproduce this glitch, turning off all of the plugins again (except vundle and nerdtree) and commenting out all autocmds and nerdtree options, except `NERDTreeQuitOnOpen=1` as it’s required to reproduce. It’s still there, in 5.3.1.

I was wrong at first place. I downgraded NERDTree to 5.2.9 and at the end of that day I reproduced the glitch just by working. It was much harder to reproduce however. Then, I downgraded to 5.2.8 and the glitch disappered. So, I’m using 5.2.8 for 11 days already and I’m not seeing it anymore.

I know this this glitch is hard to reproduce but I thought it would be good to report it. Maybe someone is also facing this wierd issue. If you cannot reproduce it, feel free to close this issue. I’m pretty happy with 5.2.8.
